### DISCUSSION TITLE
Why the experimental accuracy is so much higher than the paper ?  为什么实验精度比论文高这么多

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmrotate#625] [Q&A] [answered]
Why the experimental accuracy is so much higher than the paper
为什么实验精度比论文高这么多?

![3881c9b7733aa5441687cb5f69fe21d](https://user-images.githubusercontent.com/25839884/202352686-7ffbecb3-b549-493f-a89e-a6ce32133bd0.png)

![d438431d3622092436ae1da657064b1](https://user-images.githubusercontent.com/25839884/202352696-666cb9bd-1fee-49d6-a818-2e1939be57a7.png)

**Comment by yangxue0827** [comment, marked as answer]
This is the mAP of the trainval set, not the test set.

**Reply by Chenglu840** [reply]
如果要测试模型精度，跟论文列表差不多的，应该怎么做呢

**Reply by yangxue0827** [reply]
mmrotate并不是他们官方实现，我们只是提供了一个公平的比较环境。对于早期的方法，mmrotate是很轻松能超过他们的，对于近期的方法，可以跑他们官方开源的代码来进行性能对齐。如果没有开源我们也没法保证mmrotate的精度可以对齐，因为我们不知道他实际用了什么tricks。

**Reply by Chenglu840** [reply]
那我想要改模型的话，不能在mmrotate上直接改吧，应该改论文源码的吧

**Reply by yangxue0827** [reply]
可以基于mmrotate上改的，只要你的实验对比结果（消融实验）都是来自mmrotate就行，最后sota实验可以引用原文的。
